directory/cli/src/list_command.py

```diff
@@ -72,7 +72,7 @@ def do(
     if not all([ipa_find_command, field_configs]):
         raise TypeError
 
-    item_dicts = ipa_utils.ipa_find(ipa_find_command, ipa_find_args, all_fields=all_fields)
+    item_dicts = ipa_utils.ipa_find(command, args, all_fields=all_fields)
```

Comment:
Does this work? Where are `command` and `args` defined? I believe you can just reset this file to be identical to before this PR was made now.

Reply:
You're correct, this is why you review things for me 😛